web/tests/help-page.spec.ts

```diff
@@ -0,0 +1,62 @@
+//Tests the help page
+import { test, expect } from '@playwright/test';
+import { TableBrowser, Workspace } from './lynxkite';
+
+let workspace: Workspace;
+
+async function checkExactlyOneCurrent() {
+    // There is only one highlighted item:
+    await expect(await workspace.page.locator('span.find-highlight-current').count()).toBe(1);
+}
+
+async function checkCurrentHighlight(expectedPos) {
+    await checkExactlyOneCurrent();
+    // Check correct item:
+    const cssQuery = 'span.find-highlight.find-highlight-current.find-highlight-' + expectedPos;
+    await expect(workspace.page.locator(cssQuery)).toBeVisible();
+}
+
+test.beforeAll(async ({ browser }) => {
+    workspace = await Workspace.empty(browser);
+});
+
+test('search box in help', async () => {
+    await workspace.page.goto('#/help');
+    const field = await workspace.page.locator('#find-in-page-text');
+    await expect(field).toBeVisible;
+    // Search for the phrase "user".
+    await field.fill('user');
+    // Expect more than 5 matches.
+    await workspace.page.locator('span.find-highlight').first().waitFor();
```

Comment:
```suggestion
    await expect(workspace.page.locator('span.find-highlight').first()).toBeVisible();
```

This waits too, but also adds an assertion and it's more like all the other checks.